examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -78,7 +78,7 @@ extern "C" {
 #include "sl_si91x_trng.h"
 #define TRNGKEY_SIZE 4
 #endif // SLI_SI91X_MCU_INTERFACE
-}
+} // extern "C" {
```

Comment:
Those comments are valuable for #If #endif statement but not so much for bracket that are easily trackable. Please remove